.github/workflows/test-full-sync.yml

```diff
@@ -58,6 +58,41 @@ jobs:
         with:
           short-length: 7
 
+      - name: Authenticate to Google Cloud
+        id: auth
+        uses: google-github-actions/auth@v0.7.0
+        with:
+          workload_identity_provider: 'projects/143793276228/locations/global/workloadIdentityPools/github-actions/providers/github-oidc'
+          service_account: 'github-service-account@zealous-zebra.iam.gserviceaccount.com'
+          token_format: 'access_token'
+
+      # If we have accidentally changed the format, but not changed the version,
+      # we want to run with the old cached state, so other jobs fails.
```

Comment:
This comment does not apply to the full sync test, because it always starts from genesis.

Reply:
Fixed by: https://github.com/ZcashFoundation/zebra/pull/4074/commits/8f24f0f0dc62c307adc43d74e283e27950f9abe4